internal/request.go

```diff
@@ -50,7 +50,15 @@ func (s *SPOA) processRequest(msg spoe.Message) ([]spoe.Action, error) {
 			var ok bool
 			app, ok = s.applications[arg.Value.(string)]
 			if !ok {
-				return nil, fmt.Errorf("application %q not found", arg.Value.(string))
+				if len(s.defaultApplication) > 0 {
+					fmt.Printf("application %q not found, using default application %q\n", arg.Value.(string), s.defaultApplication)
```

Comment:
I imagine this might be a very noisy log, especially for people using `hdr(host)` as the app name since every request would trigger this message. Maybe it could be `app.logger.Debug` instead?

Reply:
I second that, something like this after reassigning `app` var after the inner if stmt: `app.logger.Debug("application not found, using default", zap.Any("application", arg.Value), zap.String("default", s.defaultApplication))`

Reply:
Thank you @mac-chaffee for idea and @jcchavezs for solution. Updated message severity.